### ISSUE TITLE
Terrastories not working out of the box

### ISSUE THREAD

**Issue by luandro** [digidem/edt-offline#18]
- [x] Terrastories still needs manual migration
- [x] Check if Terrastories is setup with tile-server

**Comment by luandro** [comment]
Current error using "terrastories/terrastories:latest" image:
```
/api/vendor/bundle/ruby/2.7.0/gems/activerecord-6.1.7.2/lib/active_record/database_configurations/connection_url_resolver.rb:25:in `initialize': Database URL cannot be empty (RuntimeError)
```

**Comment by luandro** [comment]
Solved by #48
